pkg/util/merr/utils.go

```diff
@@ -299,6 +299,13 @@ func IsHealthyOrStopping(stateCode commonpb.StateCode) error {
 	return CheckHealthy(stateCode)
 }
 
+func AnalyzeComponentStateResp(role string, nodeID int64, resp *milvuspb.ComponentStates, err error) error {
+	if err != nil {
+		return errors.Wrap(err, "service is unhealthy")
```

Comment:
the error handling with a new function doesn't make sence. and the error message doesn't show node id and role as important information

Reply:
This method is used to check the component state, with detailed information included in the check results when invoking the AnalyzeComponentStateResp methods.